### ISSUE TITLE
Keep language label in Publish menu even when dropping below threshold

### ISSUE THREAD

**Issue by direwolf420** [bluemagic123/tModLoader#3977]
### Do you intend to personally contribute/program this feature?

No

### I would like to see this change made to improve my experience with

Mod capability as a Modder

### Description

Short summary of the current situation when publishing mods with localizations:
If you have a language with 75% of keys translated, when you enter the Publish menu, it will auto-select the language label/tag.
If you have a language with less than 75%, it will not be selected.
In general, tags set for a mod from before will populate the Publish menu. This breaks the expected behavior, and modders may forget to re-label their mod as localized for a particular language.

https://github.com/tModLoader/tModLoader/blob/4e798fae66391925ba7113dd21688c66e67a9340/patches/tModLoader/Terraria/GameContent/UI/States/WorkshopPublishInfoStateForMods.TML.cs#L154-L157

### What does this proposal attempt to solve or improve?

This can be important if translators are "delayed" in supplying localization, or the new keys are something only a fraction of the userbase will get to see (some UI option in a mod, or a separate game mode), or if the mod is so small (15 keys), that adding 6 more untranslated keys will dip below 75%. The modder then still wants to have their mod labeled as localized without manually "re-confirming" it.
 
In my personal case I have a mod that adds an item similar to Lucy the Axe. 98% of the localization for this item are part of the dialogue, which most players will not see. This caused my mod to tip just below the threshold with 74% total keys translated and thus not tagged on the language. If I didn't bother scrolling down on the UI to confirm the tags, I could've missed it and published it without that tag.

Long story short - **Proposed solution:** Instead of "un-tagging" a language if it dips below the threshold, keep it but highlight it in a different color (red for "warning" with an extra mouseover explaining the threshold, since it's not actually visible to modders currently).

**Comment by JavidPack** [comment]
I think I intentionally went with the current behavior, but if there is more feedback that would like it this way we can do that.

My main goal was to prevent mods that underwent large updates to continue to advertise that they were supported by languages that never get updated. As an extreme example, I believe a mod with 0 localization entries would count as fully localized. If a modder publishes an initial release before adding content to their mod, they might mistakenly continue publishing tagged as fully localized.

**Comment by Chicken-Bones** [comment]
I would propose setting 'unlocalized' if it drops below 50%

**Comment by JavidPack** [comment]
50% sounds good, I've updated it so that if a mod was previously tagged as localized, it'll persist the tag selection as long as it is still about 50%. This should result in most mods not losing their localized status between localization updates. Mods that add a ton of localization that takes it below 50% will still lose the tag, which seems reasonable to me.

**Comment by direwolf420** [comment]
Thanks. I'd still like a mouseover or _something_  that shows these threshold values to the modder
